invenio_records_rest/utils.py

```diff
@@ -139,7 +166,7 @@ def data(self):
                 response = make_response(jsonify(data), data['status'])
                 response.headers['Location'] = location
                 abort(response)
-            except BuildError:
+            except (BuildError, ValueError):
```

Comment:
Maybe we should make sure that the configuration is check when the extension is initialized instead of catching it runtime.